includes/class-block-patterns.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * The Block Patterns post type.
```

Comment:
@jrtashjian And its relevant taxonomies

Reply:
Fixed in 091efc4f12b2d03c7788fda5fc9e035255523d8c